compareECLFiles.cmake

```diff
@@ -608,6 +608,34 @@ add_test_compareECLFiles(CASENAME 9_3c_grpctl_msw_model2
                          REL_TOL ${rel_tol}
                          DIR model2)
 
+add_test_compareECLFiles(CASENAME 9_3d_grpctl_stw_model2
+                         FILENAME 9_GINJ_GAS_MAX_EXPORT_STW
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR model2)
+
+add_test_compareECLFiles(CASENAME 9_3d_grpctl_msw_model2
+                         FILENAME 9_GINJ_GAS_MAX_EXPORT_MSW
```

Comment:
I believe this `FILENAME` should be
```
9_3D_GINJ_GAS_MAX_EXPORT_MSW
```
instead (with `_3D` inserted between `9` and `_GINJ`.)